core/src/main/java/com/arcrobotics/ftclib/util/Timing.java

```diff
@@ -12,61 +12,115 @@
     /**
      * Class for a timer. This can be used for checking if a duration has passed, only continuing
      * if the timer has finished, and so forth.
-     *
+     * <p>
      * A more simple version of a timer better suited for quick uses rather than an
-     *  {@link ElapsedTime} object.
+     * {@link ElapsedTime} object.
      */
     public static class Timer {
         private ElapsedTime time;
         private long timerLength;
-        private long pauseTime;
+        private long pauseTime; // in nanoseconds, regardless of unit
```

Comment:
Is the point of the nanoseconds for accuracy purposes?

Reply:
`ElapsedTime` internally uses nanoseconds, and although the documentation doesn't mention it, the argument to `ElapsedTime(long startTime)` is in nanoseconds. The documentation says that the resolution is in seconds, but that's different than the unit. Since `startTime` must be in nanoseconds, in order to calculate the appropriate startTime in the `resume()` method it's convenient to have the pauseTime in nanoseconds.

Reply:
Makes sense. 👍